packages/blocks/src/api/index.js

```diff
@@ -136,6 +137,7 @@ export {
 	getBlockSupport,
 	hasBlockSupport,
 	getBlockVariations,
+	getActiveBlockVariation,
```

Comment:
Do we need to export this one?

Reply:
Ah, we probably don't! I think that's a leftover from an earlier iteration. Good spot! I'll remove it.

Reply:
9965b64f17e